wodles/aws/buckets_s3/__init__.py

```diff
@@ -0,0 +1,9 @@
+from buckets_s3 import aws_bucket
```

Comment:
Same issue as mentioned [here](https://github.com/wazuh/wazuh/pull/16071#discussion_r1102846050). It would generate an import problem due to the existence of `.../wodles/gcloud/buckets`